src/api/faq/faq.test.js

```diff
@@ -1,11 +1,112 @@
 const request = require('supertest');
+const mongoose = require('mongoose');
 
 const app = require('../../app');
+const FAQ = require('./faq.model');
+
+const ids = {
+  invalid: 'invalid_id',
+  nonexistant: 'aaaaaaaaaaaaaaaaaaaaaaaa',
+};
+
+async function prePopulate() {
+  const question = 'Populate Question';
+  const answer = 'Populate answer';
+
+  const { _id } = await new FAQ({ question, answer }).save();
+  ids.valid = _id;
+}
+
+beforeAll(prePopulate);
+afterAll(() => FAQ.deleteMany({}).then(() => mongoose.connection.close()));
 
 describe('GET /faq', () => {
   it('Should respond with a 200 status code',
     () => request(app)
       .get('/faq')
       .expect('Content-Type', /json/)
       .expect(200));
+  it('Should respond with array.',
+    () => request(app)
+      .get('/faq')
+      .expect(200)
+      .then((res) => {
+        expect(res.body)
+          .toEqual(expect.any(Array));
```

Comment:
I would also add a check that the array length is greater than 0. This can be in this test or a separate test.